.svglintrc.js

```diff
@@ -473,6 +473,43 @@ module.exports = {
                 ignoreIcon(reporter.name, iconPath, $);
               }
             }
+          },
+          function(reporter, $, ast) {
+            reporter.name = "path-format";
+
+            const iconPath = $.find("path").attr("d");
+            if (!updateIgnoreFile && isIgnored(reporter.name, iconPath)) {
+              return;
+            }
```

Comment:
Do we want to keep this around for every reporter? Given that there are currently no problems I don't think we ever want this rule to be ignored :thinking: